scoring.js

```diff
@@ -0,0 +1,52 @@
+const passingScore = (player) => {
+  const yards = player.stats.passing.yards / 25
+  const touchdowns = player.stats.passing.touchdowns * 6
+  const interceptions = player.stats.passing.interceptions * -3
+
+  return yards + touchdowns + interceptions
+}
+
+const rushingScore = (player) => {
+  const yards = player.stats.rushing.yards / 10
+  const touchdowns = player.stats.rushing.touchdowns * 6
+  const fumbles = player.stats.rushing.fumbles * -3
+
+  return yards + touchdowns + fumbles
+}
+
+const returnScore = (player) => {
+  const kickReturnYards = player.stats.return.kickreturn.yards / 15
+  const kickReturnTouchdowns = player.stats.return.kickreturn.touchdowns * 6
+  const kickReturnFumbles = player.stats.return.kickreturn.fumbles * -3
+  const puntReturnYards = player.stats.return.puntreturn.yards / 15
+  const puntReturnTouchdowns = player.stats.return.puntreturn.touchdowns * 6
+  const puntReturnFumbles = player.stats.return.puntreturn.fumbles * -3
+
+  return kickReturnYards + kickReturnTouchdowns + kickReturnFumbles + puntReturnYards + puntReturnTouchdowns + puntReturnFumbles
+}
+
+const receivingScore = (player) => {
```

Comment:
`calculateReceivingScore` would be slightly better naming. Function names tend to be action based since they do things. `receivingScore` sounds more like a variable.